lib/beaker/hypervisor/aws_sdk.rb

```diff
@@ -405,7 +400,12 @@ def launch_all_nodes
           instance = create_instance(host, ami_spec, nil)
           instances.push({:instance => instance, :host => host})
         end
+        # Add metadata tags to each instance
```

Comment:
We did the f5 release with manual qa, so no it's pushed now, but I didn't want it to linger too long.

I'll get the PR updated for v2 as well.